<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="ru_RU">
<context>
    <name>AbstractLink</name>
    <message>
        <location filename="../sources/data_source/links/abstract_link.cpp" line="58"/>
        <source>The connection was refused by the peer (or timed out)</source>
        <translation>Соединение было отклонено или время вышло</translation>
    </message>
    <message>
        <location filename="../sources/data_source/links/abstract_link.cpp" line="61"/>
        <source>The remote host closed the connection</source>
        <translation>Удаленный хост закрыл соединение</translation>
    </message>
    <message>
        <location filename="../sources/data_source/links/abstract_link.cpp" line="64"/>
        <source>The host address was not found</source>
        <translation>Адрес хоста не найден</translation>
    </message>
    <message>
        <location filename="../sources/data_source/links/abstract_link.cpp" line="67"/>
        <source>The socket operation failed because the application lacked the required privileges</source>
        <translation>Операция сокета не удалась, потому что приложение не обладало требуемыми привилегиями</translation>
    </message>
    <message>
        <location filename="../sources/data_source/links/abstract_link.cpp" line="71"/>
        <source>The local system ran out of resources (e.g., too many sockets)</source>
        <translation>В локальной системе закончились ресурсы (например, слишком много сокетов)</translation>
    </message>
    <message>
        <location filename="../sources/data_source/links/abstract_link.cpp" line="74"/>
        <source>The socket operation timed out</source>
        <translation>Вышло время операции сокета</translation>
    </message>
    <message>
        <location filename="../sources/data_source/links/abstract_link.cpp" line="77"/>
        <source>The datagram was larger than the operating system&apos;s limit</source>
        <translation>Дейтаграмма была больше, чем ограничение операционной системы</translation>
    </message>
    <message>
        <location filename="../sources/data_source/links/abstract_link.cpp" line="80"/>
        <source>An error occurred with the network</source>
        <translation>Произошла ошибка с сетью</translation>
    </message>
    <message>
        <location filename="../sources/data_source/links/abstract_link.cpp" line="83"/>
        <source>The address specified is already in use and was set to be exclusive</source>
        <translation>Указанный адрес уже используется и был установлен как эксклюзивный</translation>
    </message>
    <message>
        <location filename="../sources/data_source/links/abstract_link.cpp" line="86"/>
        <source>The address specified does not belong to the host</source>
        <translation>Указанный адрес не принадлежит хосту</translation>
    </message>
    <message>
        <location filename="../sources/data_source/links/abstract_link.cpp" line="89"/>
        <source>The requested socket operation is not supported by the local operating system</source>
        <translation>Запрошенная операция сокета не поддерживается локальной операционной системой</translation>
    </message>
    <message>
        <location filename="../sources/data_source/links/abstract_link.cpp" line="93"/>
        <source>An operation was attempted while the socket was in a state that did not permit it</source>
        <translation>Операция была предпринята, когда сокет находился в состоянии, которое её не разрешало</translation>
    </message>
    <message>
        <location filename="../sources/data_source/links/abstract_link.cpp" line="97"/>
        <source>A temporary error occurred</source>
        <translation>Произошла временная ошибка</translation>
    </message>
</context>
<context>
    <name>BluetoothLink</name>
    <message>
        <location filename="../sources/data_source/links/bluetooth_link.cpp" line="84"/>
        <source>Could not find the remote host</source>
        <translation>Не удалось найти удаленный хост</translation>
    </message>
    <message>
        <location filename="../sources/data_source/links/bluetooth_link.cpp" line="87"/>
        <source>Could not find the service UUID(%1) on remote host</source>
        <translation>Не удалось найти UUID службы (% 1) на удаленном хосте</translation>
    </message>
    <message>
        <location filename="../sources/data_source/links/bluetooth_link.cpp" line="90"/>
        <source>Attempt to read or write from socket returned an error</source>
        <translation>Попытка чтения или записи из сокета возвратила ошибку</translation>
    </message>
    <message>
        <location filename="../sources/data_source/links/bluetooth_link.cpp" line="93"/>
        <source>The Protocol is not supported on this platform</source>
        <translation>Протокол не поддерживается на этой платформе</translation>
    </message>
    <message>
        <location filename="../sources/data_source/links/bluetooth_link.cpp" line="96"/>
        <source>An operation was attempted while the socket was in a state that did not permit it</source>
        <translation>Операция была предпринята, когда сокет находился в состоянии, которое её не разрешало</translation>
    </message>
    <message>
        <location filename="../sources/data_source/links/bluetooth_link.cpp" line="106"/>
        <source>An unknown error has occurred</source>
        <translation>Произошла неизвестная ошибка</translation>
    </message>
</context>
<context>
    <name>BluetoothService</name>
    <message>
        <location filename="../sources/domain/bluetooth/bluetooth_service.cpp" line="96"/>
        <location filename="../sources/domain/bluetooth/bluetooth_service.cpp" line="101"/>
        <location filename="../sources/domain/bluetooth/bluetooth_service.cpp" line="106"/>
        <location filename="../sources/domain/bluetooth/bluetooth_service.cpp" line="112"/>
        <location filename="../sources/domain/bluetooth/bluetooth_service.cpp" line="118"/>
        <location filename="../sources/domain/bluetooth/bluetooth_service.cpp" line="124"/>
        <source>Bluetooth service</source>
        <translation>Сервис Bluetooth</translation>
    </message>
    <message>
        <location filename="../sources/domain/bluetooth/bluetooth_service.cpp" line="97"/>
        <source>The Bluetooth adaptor is powered off</source>
        <translation>Адаптер Bluetooth отключен</translation>
    </message>
    <message>
        <location filename="../sources/domain/bluetooth/bluetooth_service.cpp" line="102"/>
        <source>Writing or reading from the device resulted in an error</source>
        <translation>Запись или чтение с устройства привели к ошибке</translation>
    </message>
    <message>
        <location filename="../sources/domain/bluetooth/bluetooth_service.cpp" line="107"/>
        <source>The passed local adapter address does not match the physical adapter address of any local Bluetooth device</source>
        <translation>Адрес локального адаптера не соответствует адресу физического адаптера любого локального устройства Bluetooth</translation>
    </message>
    <message>
        <location filename="../sources/domain/bluetooth/bluetooth_service.cpp" line="113"/>
        <source>Device discovery is not possible or implemented on the current platform</source>
        <translation>Обнаружение устройства невозможно или не реализовано на текущей платформе</translation>
    </message>
    <message>
        <location filename="../sources/domain/bluetooth/bluetooth_service.cpp" line="119"/>
        <source>One of the requested discovery methods is not supported by the current platform</source>
        <translation>Один из запрошенных методов обнаружения не поддерживается текущей платформой</translation>
    </message>
    <message>
        <location filename="../sources/domain/bluetooth/bluetooth_service.cpp" line="125"/>
        <source>An unknown error has occurred</source>
        <translation>Произошла неизвестная ошибка</translation>
    </message>
</context>
<context>
    <name>CommunicationService</name>
    <message>
        <location filename="../sources/domain/communication/communication_service.cpp" line="217"/>
        <location filename="../sources/domain/communication/communication_service.cpp" line="262"/>
        <location filename="../sources/domain/communication/communication_service.cpp" line="272"/>
        <source>Link</source>
        <translation>Связь</translation>
    </message>
    <message>
        <location filename="../sources/domain/communication/communication_service.cpp" line="218"/>
        <source>Connection established</source>
        <translation>Соединение установлено</translation>
    </message>
    <message>
        <location filename="../sources/domain/communication/communication_service.cpp" line="218"/>
        <source>Connection lost</source>
        <translation>Потеря связи</translation>
    </message>
    <message>
        <location filename="../sources/domain/communication/communication_service.cpp" line="258"/>
        <source>Unknown</source>
        <translation>Неизвестно</translation>
    </message>
    <message>
        <location filename="../sources/domain/communication/communication_service.cpp" line="259"/>
        <source>Switched on MAVLINK v.1</source>
        <translation>Переключена на MAVLINK v.1</translation>
    </message>
    <message>
        <location filename="../sources/domain/communication/communication_service.cpp" line="260"/>
        <source>Switched on MAVLINK v.2</source>
        <translation>Переключена на MAVLINK v.2</translation>
    </message>
</context>
<context>
    <name>DefaultParamsMigration</name>
    <message>
        <location filename="../sources/data_source/db_manager/migrations/alter_link_description_migration.cpp" line="32"/>
        <location filename="../sources/data_source/db_manager/migrations/default_params_migration.cpp" line="26"/>
        <source>UDP Link</source>
        <translation>UDP Связь</translation>
    </message>
    <message>
        <location filename="../sources/data_source/db_manager/migrations/alter_link_description_migration.cpp" line="40"/>
        <source>Radiomodem</source>
        <translation>Радиомодем</translation>
    </message>
    <message>
        <location filename="../sources/data_source/db_manager/migrations/default_params_migration.cpp" line="34"/>
        <source>Serial Link</source>
        <translation>Послед. связь</translation>
    </message>
    <message>
        <location filename="../sources/data_source/db_manager/migrations/default_params_migration.cpp" line="44"/>
        <source>MAV 1</source>
        <translation>БЛА 1</translation>
    </message>
</context>
<context>
    <name>SerialLink</name>
    <message>
        <location filename="../sources/data_source/links/serial_link.cpp" line="92"/>
        <source>Attempting to open an non-existing device %1</source>
        <translation>Попытка открыть несуществующее устройство %1</translation>
    </message>
    <message>
        <location filename="../sources/data_source/links/serial_link.cpp" line="95"/>
        <source>Attempting to open an already opened device %1 or a user not having enough permission to open</source>
        <translation>Попытка открыть уже открытое устройство% 1 или пользователь, не имет прав на открытие</translation>
    </message>
    <message>
        <location filename="../sources/data_source/links/serial_link.cpp" line="99"/>
        <source>Attempting to open an already opened device in this object</source>
        <translation>Попытка открыть уже открытое устройство в этом объекте</translation>
    </message>
    <message>
        <location filename="../sources/data_source/links/serial_link.cpp" line="102"/>
        <source>Operation is executed that can only be successfully performed if the device is open</source>
        <translation>Выполняется операция, которая может быть успешно выполнена только в том случае, если устройство открыто</translation>
    </message>
    <message>
        <location filename="../sources/data_source/links/serial_link.cpp" line="106"/>
        <source>An I/O error occurred while writing the data</source>
        <translation>Ошибка ввода-вывода при записи данных</translation>
    </message>
    <message>
        <location filename="../sources/data_source/links/serial_link.cpp" line="109"/>
        <source>An I/O error occurred while reading the data</source>
        <translation>Ошибка ввода-вывода при чтении данных</translation>
    </message>
    <message>
        <location filename="../sources/data_source/links/serial_link.cpp" line="112"/>
        <source>An I/O error occurred when a resource becomes unavailable</source>
        <translation>Ошибка ввода-вывода произошла, когда ресурс стал недоступным</translation>
    </message>
    <message>
        <location filename="../sources/data_source/links/serial_link.cpp" line="117"/>
        <source>The requested device operation is not supported or prohibited by the operating system</source>
        <translation>Запрошенная операция устройства не поддерживается или запрещена операционной системой</translation>
    </message>
    <message>
        <location filename="../sources/data_source/links/serial_link.cpp" line="121"/>
        <source>A timeout error occurred</source>
        <translation>Произошла ошибка времени ожидания</translation>
    </message>
    <message>
        <location filename="../sources/data_source/links/serial_link.cpp" line="125"/>
        <source>An unidentified error occurred</source>
        <translation>Произошла неопознанная ошибка</translation>
    </message>
</context>
<context>
    <name>Units</name>
    <message>
        <location filename="../sources/utils/units.cpp" line="70"/>
        <source>m/s</source>
        <translation>м/с</translation>
    </message>
    <message>
        <location filename="../sources/utils/units.cpp" line="71"/>
        <source>km/h</source>
        <translation>км/ч</translation>
    </message>
    <message>
        <location filename="../sources/utils/units.cpp" line="72"/>
        <source>knot</source>
        <translation>уз</translation>
    </message>
    <message>
        <location filename="../sources/utils/units.cpp" line="73"/>
        <source>mph</source>
        <translation>мл/ч</translation>
    </message>
    <message>
        <location filename="../sources/utils/units.cpp" line="82"/>
        <source>m</source>
        <translation>м</translation>
    </message>
    <message>
        <location filename="../sources/utils/units.cpp" line="83"/>
        <source>km</source>
        <translation>км</translation>
    </message>
    <message>
        <location filename="../sources/utils/units.cpp" line="84"/>
        <source>ft</source>
        <translation>фут</translation>
    </message>
    <message>
        <location filename="../sources/utils/units.cpp" line="85"/>
        <source>mi</source>
        <translation>мил</translation>
    </message>
</context>
<context>
    <name>VehicleService</name>
    <message>
        <location filename="../sources/domain/vehicle/vehicle_service.cpp" line="150"/>
        <source>MAV %1</source>
        <translation>БЛА %1</translation>
    </message>
</context>
<context>
    <name>data_source::DbMigrator</name>
    <message>
        <location filename="../sources/data_source/db_manager/db_migrator.cpp" line="29"/>
        <source>Migrate error: version: %1, error: %2</source>
        <translation>Ошибка миграции версии %1, ошибка: %2</translation>
    </message>
    <message>
        <location filename="../sources/data_source/db_manager/db_migrator.cpp" line="37"/>
        <source>Migration version: %1</source>
        <translation>Версия миграции: %1</translation>
    </message>
    <message>
        <location filename="../sources/data_source/db_manager/db_migrator.cpp" line="50"/>
        <source>Drop migration error: version: %1, error: %2</source>
        <translation>Ошибка отката миграции версии %1, ошибка: %2</translation>
    </message>
    <message>
        <location filename="../sources/data_source/db_manager/db_migrator.cpp" line="77"/>
        <source>Error: </source>
        <translation>Ошибка: </translation>
    </message>
    <message>
        <location filename="../sources/data_source/db_manager/db_migrator.cpp" line="88"/>
        <source>Reset migrations</source>
        <translation>Сброс миграций</translation>
    </message>
    <message>
        <location filename="../sources/data_source/db_manager/db_migrator.cpp" line="95"/>
        <source>Establish migration %1</source>
        <translation>Установлена миграция %1</translation>
    </message>
    <message>
        <location filename="../sources/data_source/db_manager/db_migrator.cpp" line="100"/>
        <source>Migrate from %1 to %2</source>
        <translation>Миграция с версии %1 на %2</translation>
    </message>
</context>
</TS>
